1-org/projects.tf

```diff
@@ -97,7 +97,7 @@ module "interconnect" {
 
   labels = {
     environment      = "prod"
-    application_name = "prj-interconnect"
+    application_name = "interconnect"
```

Comment:
if the label is fixed, the search for the label needs to be fixed too: 
https://github.com/terraform-google-modules/terraform-example-foundation/blob/e7ef9c22071fd562aba400ff0026cfb60a738a30/3-networks/modules/dedicated_interconnect/main.tf#L22